docs/Architecture_and_Design.md

```diff
@@ -0,0 +1,93 @@
+Why CORIO (KOR-I-O) 
```

Comment:
Use headers [github-link](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings)

Reply:
Done